packages/block-editor/src/store/defaults.js

```diff
@@ -139,5 +136,12 @@ export const SETTINGS_DEFAULTS = {
 	availableLegacyWidgets: {},
 	hasPermissionsToManageWidgets: false,
 	__experimentalCanUserUseUnfilteredHTML: false,
-};
+	snap: {
+		// The snap grid lines that the image will snap to
+		// those should be overridden by themes later
+		gridStops: [ 0, 75, 150, 225, 300, 375, 450, 525, 600, 675 ],
 
+		// snap gap is sued to snap the image to the line when it sizes approach this point
+		gap: 20,
+	},
```

Comment:
What would be the perfect API here? I feel the "gap" shouldn't be part of the API and just an implementation detail. and maybe instead of `gridStops`, we'd just provide the number of columns in the main area?

I also think the same config should be possible to override in  `InnerBlocks` I feel each inner block area could potentially define its own "grid".